### PR TITLE
James1/login

### PR DESCRIPTION
**Pull request by gersbach**
## Summary
What does this PR contribute?

Styling and some functionality for the login and verification screen.

## Test Plan
How should the person reviewing this PR test it out? For example, what screens in the app should they go to?

Please take a look at the styling and make sure everything looks similar to what's on figma. Also please review all functionality such as the inputs for the verification code, and make sure that the PR goes to the root. Also, please check to see if there is latency on any part of the styling modifications.  

## Notes
(You can leave this empty if you don't have anything.)

## Screenshots and Tests
If your feature is visual, attach before/after screenshots and ensure that tests still pass

### CODE REVIEW

File: src/components/ImageUpload.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/* eslint-disable operator-linebreak */
+import React, { useState, useEffect } from 'react';
```
Comment (GuessingMan) [inline src/components/ImageUpload.js:2]: why is ImageUpload.js a part of this PR? Is this a mistake or was it necessary?

File: src/navigation/index.js [hunk 2]
```diff
@@ -58,7 +58,7 @@ function RootNavigator() {
       <Screen
         name="Login"
         options={{ headerShown: false, gestureEnabled: false }}
-        component={LoginScreen}
+        component={LoginScreen} // LoginScreen
```
Comment (GuessingMan) [inline src/navigation/index.js:61]: Please delete comments and revert any temporary changes

File: src/database/firebase.js [hunk 3]
```diff
@@ -8,6 +8,12 @@ import {
   // eslint-disable-next-line import/no-unresolved
 } from '@env';
 import firebase from 'firebase/compat/app';
+import {
```
Comment (GuessingMan) [inline src/database/firebase.js:11]: more image upload, was this because of the previous PR you worked on?

File: src/screens/AddScreen.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { useState, useEffect } from 'react';
 import { StyleSheet } from 'react-native';
 import { Switch, Title, TextInput, Button } from 'react-native-paper';
 
+import ImageUpload from '../components/ImageUpload';
```
Comment (GuessingMan) [inline src/screens/AddScreen.js:7]: also image upload

File: src/screens/LoginScreen.js [hunk 5]
```diff
@@ -129,48 +187,63 @@ export default function LoginScreen({ navigation }) {
...
+            </Text>
+          ) : null}
+        </ViewContainer>
+
+        <VerifyButton
+          phoneNumber={phoneNumber}
+          setMessage={setMessage}
+          setVerificationId={setVerificationId}
+          recaptchaVerifier={recaptchaVerifier}
+
```
Comment (GuessingMan) [inline src/screens/LoginScreen.js:241]: delete this space

File: src/screens/VerificationScreen.js [hunk 6]
```diff
@@ -1,79 +1,282 @@
...
+
+    );
+  }
+  const input1 = useRef(null);
+  const input2 = useRef(null);
+  const input3 = useRef(null);
+  const input4 = useRef(null);
+  const input5 = useRef(null);
+  const input6 = useRef(null);
+  const currentInput = useRef(null)
```
Comment (GuessingMan) [inline src/screens/VerificationScreen.js:193]: this variable doesnt seem to be used

File: src/screens/VerificationScreen.js [hunk 7]
```diff
@@ -1,79 +1,282 @@
...
+
+  const { verificationId } = true; // route.params;
```
Comment (GuessingMan) [inline src/screens/VerificationScreen.js:195]: change this back?

File: src/screens/VerificationScreen.js [hunk 8]
```diff
@@ -1,79 +1,282 @@
...
+            />
+            <VerficationFields
+              inputRef={input6}
+              number={6}
+              inputNextRef={input6}
+            />
+          </View>
+          <View style={styles.rowContainer}>
+            <Text >
+              {"\Didn't get a code? "}
```
Comment (GuessingMan) [inline src/screens/VerificationScreen.js:248]: is the backslash supposed to be there in the front? or the space in the end?

File: src/screens/VerificationScreen.js [hunk 9]
```diff
@@ -1,79 +1,282 @@
...
+        style={styles.button}
         title="Confirm Verification Code"
         disabled={!verificationId}
         onPress={async () => {
+          const verifcationCode = verifCode.join();
           try {
             const credential = firebase.auth.PhoneAuthProvider.credential(
               verificationId,
-              verificationCode,
+              verifcationCode,
```
Comment (GuessingMan) [inline src/screens/VerificationScreen.js:172]: i think you misspelled "verification"

File: src/screens/VerificationScreen.js [hunk 10]
```diff
@@ -1,79 +1,282 @@
...
+      verifCode[number] = inputNumber;
+      setVerifCode(verifCode);
+      inputNextRef.current.focus();
+    };
+
+    const onFocus = () => setFocused(true)
+    const onBlur = () => setFocused(false)
+
+    return (
+      <>
```
Comment (GuessingMan) [inline src/screens/VerificationScreen.js:144]: whats the point of <>?

Comment (GuessingMan) [reply src/screens/VerificationScreen.js:144]: and </> ?

File: env.d.ts [hunk 11]
```diff
@@ -0,0 +1,8 @@
+declare module '@env' {
```
Comment (GuessingMan) [inline env.d.ts:1]: is this supposed to be pushed? im not sure if we're supposed to be able to see env files (will ask kabir)

**Review by GuessingMan — CHANGES_REQUESTED** [review]
most changes are about leftover comments and an accidental inclusion of ImageUpload. when i try to open the app i run into these errors that ive screen shotted
IMG_7952
IMG_7953
:

File: env.d.ts (hunk 11 above)
Comment (gersbach) [reply env.d.ts:1]: I added a gitignore! Sorry!

Comment (gersbach) [reply env.d.ts:1]: hmmm it was already in gitignore. please lmk if this happens again!

**Pull request closed without merging**